docs/source/peer-chaincode-devmode.rst

```diff
@@ -0,0 +1,113 @@
+::
+
+Generate the genesis block.
+::
+    configtxgen -profile SampleDevModeSolo -channelID syschannel -outputBlock genesisblock -configPath $FABRIC_CFG_PATH -outputBlock $(pwd)/sampleconfig/genesisblock
+::
+
+Start the orderer
+-----------------
+
+::
+
+    ORDERER_GENERAL_GENESISPROFILE=SampleDevModeSolo orderer
+
+The above starts the orderer with the SampleDevModeSolo profile.
+
+Start the peer in dev mode
+--------------------------
+
+::
+
+    CORE_PEER_CHAINCODELISTENADDRESS=0.0.0.0:7052 peer node start --peer-chaincodedev=true
+
+The above command starts the peer using the default MSP.
+``--peer-chaincodedev=true`` puts it in “dev” mode.
+
+Create channel ch1
+---------------------------
+
+Generate the transactions for creating the channels using the ``configtxgen``
+tool with the SampleSingleMSPChannel profile.
+
+::
+
+   configtxgen -channelID ch1 -outputCreateChannelTx ch1.tx -profile SampleSingleMSPChannel -configPath $FABRIC_CFG_PATH
+
+::
+
+    peer channel create -o 127.0.0.1:7050 -c ch1 -f ch1.tx
+
+The above command assumes the orderer is reachable at ``127.0.0.1:7050``. The orderer
+is now tracking channel ch1 with the default configuration.
+
+::
+
+    peer channel join -b ch1.block
+
+The peer has now joined channel ch1.
+
+Build the chaincode
+-----------------------------
+
+The `simple` chaincode from the `integration/chaincode` directory is used here as an example.
+
+::
+    go build -o simpleChaincode ./integration/chaincode/simple/cmd
+::
+
+Start the chaincode
+-------------------
+
+::
+    CORE_CHAINCODE_LOGLEVEL=debug CORE_PEER_ADDRESS=192.168.0.7:7052 CORE_PEER_TLS_ENABLED=false CORE_CHAINCODE_ID_NAME=mycc:1.0 ./simpleChaincode -peer.address 192.168.0.7:7052
+
+This starts the chaincode and connects it to the peer. Look at the peer and chaincode logs to determine if the registration was successful.
+- Note: When DevMode is enabled, `CORE_CHAINCODE_ID_NAME` must be set to `{ChaincodeName}:{ChaincodeVersion}`; hence, for this example, it is set to `mycc:1.0` here. Otherwise, the peer will not be able to find the chaincode.
+
+Approve and commit the chaincode definition
+--------------------------------------
+
+::
+    peer lifecycle chaincode approveformyorg  -o 127.0.0.1:7050 --channelID ch1 --name mycc --version 1.0 --sequence 1 --init-required --signature-policy "OR ('SampleOrg.member')" --package-id mycc:1.0
+::
+
+::
+    peer lifecycle chaincode checkcommitreadiness -o 127.0.0.1:7050 --channelID ch1 --name mycc --version 1.0 --sequence 1 --init-required --signature-policy "OR ('SampleOrg.member')"
+
+    peer lifecycle chaincode commit -o 127.0.0.1:7050 --channelID ch1 --name mycc --version 1.0 --sequence 1 --init-required --signature-policy "OR ('SampleOrg.member')" --peerAddresses 192.168.0.7:7051
+::
+
+The above steps follow the new lifecycle commands which approve chaincode in both channels, and commit the chaincode to the ledger. At this point, you can invoke and query the chaincode as usual.
```

Comment:
```suggestion
The above steps utilize the new lifecycle commands to approve the chaincode definition and then commit the chaincode definition to the ledger. At this point, you can invoke and query the chaincode as usual.
```
Not sure where "both channels" came from. :)

Reply:
🧹  I'm sure it came from the 1.4 doc on this same topic